test/units/testsuite-26.sh

```diff
@@ -79,6 +79,14 @@ systemctl list-jobs --before
 systemctl list-jobs --after --before
 systemctl list-jobs "*"
 
+# is-* verbs
+# Should return 4 for a missing unit file
+systemctl is-active not-found.service || (( $? == 4 ))
+systemctl is-failed not-found.service || (( $? == 4 ))
+systemctl is-enabled not-found.service || (( $? == 4 ))
+# return 3 when the unit exists but inactive
+systemctl is-active "$UNIT_NAME" || (( $? == 3 ))
```

Comment:
Hmm, this will also pass if the return value is 0. Why not use `;` instead of `||` ?

Reply:
Please use `assert_rc` defined in assert.sh.